cob_base_drive_chain/CMakeLists.txt

```diff
@@ -1,42 +1,29 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_base_drive_chain)
 
-
 find_package(catkin REQUIRED COMPONENTS roscpp std_msgs std_srvs sensor_msgs message_generation diagnostic_msgs cob_generic_can cob_canopen_motor cob_utilities libpcan libntcan control_msgs)
 
-#######################################
-## Declare ROS messages and services ##
-#######################################
-
+### Message Generatioin ###
 add_service_files(
   FILES
-  ElmoRecorderConfig.srv 
-  ElmoRecorderReadout.srv 
+  ElmoRecorderConfig.srv
+  ElmoRecorderReadout.srv
 )
 
-## Generate added messages and services with any dependencies listed here
 generate_messages(
-  DEPENDENCIES std_msgs
+  DEPENDENCIES
 )
 
-###################################
-## catkin specific configuration ##
-###################################
 
 catkin_package(
-    CATKIN_DEPENDS roscpp control_msgs cob_generic_can cob_canopen_motor cob_utilities libpcan std_msgs std_srvs sensor_msgs diagnostic_msgs message_runtime
-    INCLUDE_DIRS common/include
-    LIBRARIES ${PROJECT_NAME}
+    CATKIN_DEPENDS roscpp control_msgs cob_generic_can cob_canopen_motor cob_utilities libpcan std_msgs std_srvs sensor_msgs diagnostic_msgs message_runtime INCLUDE_DIRS common/include LIBRARIES ${PROJECT_NAME}
```

Comment:
Did roscompile put this into one line?
For the sake of readability, having a line for each of `CATKIN_DEPENDS`, `INCLUDE_DIRS`, `LIBRARIES` would be better, IMO.